### PR TITLE
Activate eslint ban-ts-comment

### PR DESCRIPTION
**Pull request by vilsbole**
[ch6855]

### CODE REVIEW

File: src/services/WalletService/deterministic/ledger.ts
Comment (vilsbole) [inline src/services/WalletService/deterministic/ledger.ts:112]: The original `toString()` seemed like an error.

Comment (Mrtenz) [reply src/services/WalletService/deterministic/ledger.ts:112]: I don't think it is. `v` is a regular number, not a hexadecimal string by default. In the signature it should be hexadecimal.

Comment (Mrtenz) [inline src/services/WalletService/deterministic/ledger.ts:112]: This should be investigated before merging.

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: src/services/WalletService/deterministic/ledger.ts
Comment (vilsbole) [reply src/services/WalletService/deterministic/ledger.ts]: The `.d.ts` file types it as string. https://github.com/MyCryptoHQ/MyCrypto/blob/265677604311c58cfdfbf97933112811e7cf6e2f/src/types/vendor/ledger/hw-app-eth/index.d.ts#L6

But if I change the type, other conflicts arise.
There is a mismatch in types and usage.
Since this is out of scope I reverted the change and added a todo comment.

File: src/components/NetworkSelectDropdown.tsx
Comment (vilsbole) [reply src/components/NetworkSelectDropdown.tsx:63]: solved with #3544 
waiting for merge.

File: package.json
Comment (vilsbole) [reply package.json:270]: Just to be more declarative and separate the command from the implementation (css, eslint etc)

File: src/components/NetworkSelectDropdown.tsx
Comment (vilsbole) [reply src/components/NetworkSelectDropdown.tsx:63]: merged

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**
